### ISSUE TITLE
Toggle live-search-style

### ISSUE THREAD

**Issue by mkddostro** [snapappointments/bootstrap-select#2721]
Hey,
I tried dynamically changing the live-search-style based on user input (toggle/radio). The select is properly manipulated (attribute changes from "contains" to "startsWith" or vice versa) but selectpicker does not change its behaviour (even after a refresh and/or render, manipulating _data()_ and/or _attr()_).
Is this intended to be (im)possible?

Sorry if this has been discussed before or if there is a simple solution. I couldn't find anything about it.

```javascript
$('input[name="live-search-toggle"]').change(function(){
     $('#input-filter-tags').attr('data-live-search-style', $(this).val()).selectpicker('refresh');
});
```

**Comment by NicolasCARPi** [comment]
I believe there is an issue with the refresh code given the other issues mentionning it doesn't work. Can you try the latest code from `dev` branch? I don't recall if there are some related fixes in there or not.

**Comment by mkddostro** [comment]
I forgot to mention which version I am using: 1.12.11
Pretty old, I know. The newest version throws an error for me (something about missing functions, I tried earlier but didn't save the log).

I got it to work with this beauty of chaining:
```javascript
$('#input-filter-tags').selectpicker('destroy')
                       .attr('data-live-search-style', $(this).val())
                       .data('live-search-style', $(this).val())
                       .selectpicker();
```
which may indicate an issue with "refresh" because that didn't work. But since I am on an old version I wouldn't read too much into my issue.

Still thanks a ton for replying quickly!
